src/components/Calculator.js

```diff
@@ -7,10 +7,9 @@ import Buttons from "./Buttons";
 
 const Calculator = () => {
   const [value, setValue] = React.useState(0)
+  const operators = ["X", "%", "±", "AC", "+", "-", "/", "="]
 
-  function onButtonClick(val) {
-    setValue(val)
-  }
+  const onButtonClick = (val) => setValue(prevValue => operators.includes(val) ? prevValue : val)
```

Comment:
This works, but we'd be better off not even calling `setValue` if `val` is an operator.
So something like:
```
  const onButtonClick = (val) => {
    if (!operators.includes(val) {
      setVal(val)
    }
  }